src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgreSQL.java

```diff
@@ -88,12 +88,11 @@
     private volatile FileLock lock;
     private final boolean cleanDataDirectory;
 
-    EmbeddedPostgreSQL(File parentDirectory, File dataDirectory, boolean cleanDataDirectory, Map<String, String> postgresConfig) throws IOException
+    EmbeddedPostgreSQL(File parentDirectory, File dataDirectory, boolean cleanDataDirectory, Map<String, String> postgresConfig, int port) throws IOException
```

Comment:
Consider making a second constructor with the same signature as the old one, which calls this new one. This would maintain backward compatibility and allow users not have to deal with the magic value of 0 for port.